bin/cml/runner/launch.js

```diff
@@ -23,19 +23,26 @@ const shutdown = async (opts) => {
   RUNNER_SHUTTING_DOWN = true;
 
   const { error, cloud } = opts;
-  const { name, tfResource, noRetry, reason, destroyDelay } = opts;
+  const { name, tfResource, noRetry, reason, destroyDelay, watcher } = opts;
 
   const unregisterRunner = async () => {
     if (!RUNNER) return;
 
     try {
       winston.info(`Unregistering runner ${name}...`);
       await cml.unregisterRunner({ name });
-      RUNNER && RUNNER.kill('SIGINT');
-      winston.info('\tSuccess');
     } catch (err) {
+      if (err.message.includes('is still running a job')) {
+        winston.warn(`\tCancelling shutdown: ${err.message}`);
+        return false;
+      }
+
       winston.error(`\tFailed: ${err.message}`);
     }
+
+    RUNNER && RUNNER.kill('SIGINT');
```

Comment:
```suggestion
    RUNNER.kill('SIGINT');
```
The beginning of the function starts with an `if (!RUNNER) return;` I would like us to avoid this kind of type truthy conversion execution things...

we can break this out into a separate Tech debt issue for discussion if you disagree.

Reply:
> _we can break this out into a separate Tech debt issue for discussion if you disagree_

Thanks for the deference, but I don't disagree. Let's “revisit” the logic. 😂 

![_85331442_85331441_720](https://user-images.githubusercontent.com/11387611/210031401-d6d4c925-b917-408f-ad19-cc2d2fdae398.jpg)